packages/test/test-end-to-end-tests/src/test/gc/gcStats.spec.ts

```diff
@@ -61,19 +74,39 @@ describeCompat("Garbage Collection Stats", "NoCompat", (getTestObjectProvider) =
 
 	beforeEach(async function () {
 		provider = getTestObjectProvider({ syncSummarizer: true });
-		// These tests validate the GC stats in summary by calling summarize directly on the container runtime.
+		// These tests validate the GC stats in summary by calling summarize directly on the mainContainer runtime.
```

Comment:
I think this should not say "directly on the mainContainer".  Maybe update this comment to say more about disabling summarizer heurestics and only calling it/GC explicitly, on a separate container.

Reply:
Updated the comment.